build.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+COZY_ENV_DFL=production
+
+[ -z ${COZY_ENV} ] && COZY_ENV=${COZY_ENV_DFL}
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
I'm using zsh, I don't think `BASH_SOURCE` will work with it. And I think it's dash on travis (if we want to use this script on travis later).